### DISCUSSION TITLE
When using the win10 full screen start menu, I can't see the text I typed to search for.

### DISCUSSION THREAD

**Discussion by 6Tom** [valinet/ExplorerPatcher#1117] [Q&A]
Hi friends,

I have discovered a problem and I have guessed the cause myself, but I cannot solve it on my own. I hope I can get some help.

- If I use "the full screen start menu", it has the following problem: For example, if I first press "win", type "cmd", and then press "Enter", I can open a "command prompt window", but I cannot see what I have entered.

- If I don't use "the full screen start menu", just use "the normal start menu", press "win", and then type "cmd", "Windows Search" will pop up directly, and I can see what I have entered. This is the effect I want.

I think this is because "the top priority of the full screen start menu was incorrectly set to be higher than Windows Search".

So can someone help me with that?

**Comment by Jooty75** [comment]
Happens to me too. After typing the search term, if you right click on the Windows icon, the search box will appear.
